src/emailLogin/runMigration.js

```diff
@@ -0,0 +1,88 @@
+const axios = require('axios');
```

Comment:
suggest renaming this to `generateMigrations` instead to avoid confusion

Reply:
We actually run the migrations though, just that we additionally log the insert query in case we need to revert for any reason, do you think the name is clear enough?

Reply:
my bad; i think `runMigration` is ok assuming that we don't ahve other email login related migrations